_posts/2024-09-12-using-search-latency-monitoring.markdown

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title: Using search latency monitoring at the coordinator-node-level 
+authors:
+   - davizane
+   - naarcha
+   - akjain
+   - kkhatua
+date: 2024-09-12 
+categories:
+  - technical-posts
+  - search
+excerpt: 
+meta_keywords:
+meta_description: 
```

Comment:
Please add the following meta: 

**_meta_keywords:_**  search latency, OpenSearch cluster, latency monitoring, coordinator node, search performance optimization

**_meta_description:_**  Explore how enhanced search latency monitoring capabilities at the coordinator node level can help you optimize search performance in your OpenSearch cluster.